PlanetView.cpp

```diff
@@ -321,7 +369,72 @@ void PlanetView::SecurityChanged()
     }
 }
 
+void PlanetView::TributeChanged()
+{
+    if(object && !object->GetPlanet().isEmpty())
+    {
+        double value = numeric_limits<double>::quiet_NaN();
+        if(!tribute->text().isEmpty())
+            value = tribute->text().toDouble();
+        Planet &planet = mapData.Planets()[object->GetPlanet()];
+        if(planet.Tribute() != value)
```

Comment:
Note: this won't work with NaN values, because any comparison involving NaN always returns false.